src/messages/layout.js

```diff
@@ -21,9 +21,9 @@ export const BAD_ZIPFILE = {
 export const TYPE_NO_MANIFEST_JSON = {
   code: 'TYPE_NO_MANIFEST_JSON',
   message: _('manifest.json was not found'),
-  description: _(oneLine`A manifest.json in the root of the
-     extension was not found. See: https://mzl.la/2r2McKv for more on
-     packaging`),
+  description: _(oneLine`A manifest.json in the root of the extension was not
+    found. Check that your add-on code is not nested in a subfolder.
+    See: https://mzl.la/2r2McKv for more on packaging.`),
```

Comment:
When we're updating it… we're first saying `extension` but then referring to an `add-on`, I think we should unify this a bit?

Reply:
Updated messaging.